### ISSUE TITLE
html-formatter v19 does not include assets

### ISSUE THREAD

**Issue by mpkorstanje** [cucumber/html-formatter#65]
```
$ wget -q https://repo1.maven.org/maven2/io/cucumber/html-formatter/19.0.0/html-formatter-19.0.0.jar
$ unzip -l html-formatter-19.0.0.jar 
Archive:  html-formatter-19.0.0.jar
  Length      Date    Time    Name
---------  ---------- -----   ----
        0  2022-03-25 10:06   META-INF/
      329  2022-03-25 10:06   META-INF/MANIFEST.MF
        0  2022-03-25 10:06   io/
        0  2022-03-25 10:06   io/cucumber/
        0  2022-03-25 10:06   io/cucumber/htmlformatter/
        0  2022-03-25 10:06   META-INF/maven/
        0  2022-03-25 10:06   META-INF/maven/io.cucumber/
        0  2022-03-25 10:06   META-INF/maven/io.cucumber/html-formatter/
     1814  2022-03-25 10:06   io/cucumber/htmlformatter/Main.class
     4164  2022-03-25 10:06   io/cucumber/htmlformatter/MessagesToHtmlWriter.class
     2789  2022-03-25 10:05   META-INF/maven/io.cucumber/html-formatter/pom.xml
       61  2022-03-25 10:06   META-INF/maven/io.cucumber/html-formatter/pom.properties
---------                     -------
     9157                     12 files
```

This should include:

```
io/cucumber/htmlformatter/cucumber-html.css
io/cucumber/htmlformatter/cucumber-html.js
io/cucumber/htmlformatter/index.mustache.html
```	

It looks like this this part of the makefile is not included in the github release process:

https://github.com/cucumber/html-formatter/blob/main/Makefile#L26-L36

**Comment by aurelien-reeves** [comment]
The ruby gem properly includes assets.
The `prepare` action is also properly part of the `release-java` workflow.

The issue must be related to `actions/setup-java@v3` or `uses: cucumber/action-publish-mvn@v1.1.0`

**Comment by mpkorstanje** [comment]
The Java action cleans before making a release. So copying to `target` doesn't work.

https://github.com/cucumber/action-publish-mvn/blob/main/action.yaml#L23

**Comment by mpkorstanje** [comment]
There is not I think it can be removed from `cucumber/action-publish-mvn`.

**Comment by mattwynne** [comment]
I've released v2.0.0 of https://github.com/cucumber/action-publish-mvn/ with `clean` no longer in the default maven goals.
